### ISSUE TITLE
Endpoints not working

### ISSUE THREAD

**Issue by GeorgeNakhle** [Techial/DBD-PerkBase#3]
None of the endpoints in this repo return anything, presumably because this was deployed on Heroku's free-tier which has since been disbanded. Here's the error message I'm receiving:
![error](https://user-images.githubusercontent.com/55711921/218680229-e3753eab-aaf9-4599-aa41-dc5ca3d6c5a6.png)

**Comment by Techial** [comment]
You're correct in assuming that this API was deployed on Heroku's free-tier. I'm unsure what services currently provide free deployment of Express.js applications.

One solution would be to rewrite into serverless functions.
